src/config.js

```diff
@@ -612,6 +594,26 @@ export function newConfig() {
     }
   }
 
+  function mergeConfig(obj) {
+    if (!utils.isPlainObject(obj)) {
+      utils.logError('mergeConfig input must be an object');
+      return;
+    }
+
+    const mergedConfig = Object.keys(obj).reduce((accum, data) => {
```

Comment:
'data' might be a misleading term. I think 'key' or 'property' would be more accurate. 
'data' would be a better name for the property value.

Reply:
Updated 'data' to 'key', you're right 'key' is a bit clearer